streamer/src/nonblocking/quic.rs

```diff
@@ -1264,6 +1351,36 @@ pub mod test {
         config
     }
 
+    #[cfg(any(windows, target_os = "ios"))]
+    fn make_quic_sockets() -> Vec<UdpSocket> {
+        let socket = UdpSocket::bind("127.0.0.1:0").unwrap();
+        vec![socket]
+    }
+
+    #[cfg(not(any(windows, target_os = "ios")))]
+    fn make_quic_sockets() -> Vec<UdpSocket> {
+        use std::{
+            os::fd::{FromRawFd, IntoRawFd},
+            str::FromStr,
+        };
+
+        const NUM_QUIC_SOCKETS: usize = 10;
+        (0..NUM_QUIC_SOCKETS)
+            .map(|_| {
+                let sock = socket2::Socket::new(
+                    socket2::Domain::IPV4,
+                    socket2::Type::DGRAM,
+                    Some(socket2::Protocol::UDP),
+                )
+                .unwrap();
+                sock.set_reuse_port(true).unwrap();
+                sock.bind(&SocketAddr::from_str("127.0.0.1:0").unwrap().into())
```

Comment:
I think this should bind to port 0 to give a random port for the test. but subsequent binds should then be to the same port, right? @t-nelson @lijunwangs

Reply:
Yes, I think it should use bind_to_with_config combined with bind_more_with_config.

Reply:
ok, updated to that.